mon-pix/tests/acceptance/user-dashboard-test.js

```diff
@@ -160,6 +160,19 @@ describe('Acceptance | User dashboard page', function() {
 
   });
 
+  describe('retryable-competences', function() {
+
+    beforeEach(async function() {
+      user = server.create('user', 'withEmail');
+      await authenticateByEmail(user);
+      await visit('/accueil');
+    });
+
+    it('should display recommended-competences section', function() {
+      expect(find('section[data-test-improvable-competences]')).to.exist;
```

Comment:
nit: ou un truc du genre ?
```suggestion
      expect(contains(this.intl.t('pages.dashboard.improvable-competences.subtitle'))).to.exist;
```